db/seeds.rb

```diff
@@ -1,12 +1,34 @@
 if Color.table_exists? && Color.count == 0
-  Color.create name: 'red',   hex_value: '#FF0000'
-  Color.create name: 'green', hex_value: '#00FF00'
-  Color.create name: 'blue',  hex_value: '#0000FF'
+  Color.create name: 'red',    hex_value: '#FF0000'
+  Color.create name: 'green',  hex_value: '#00FF00'
+  Color.create name: 'blue',   hex_value: '#0000FF'
+  Color.create name: 'orange', hex_value: '#FF4500'
+  Color.create name: 'gold',   hex_value: '#FFD700'
+  Color.create name: 'violet', hex_value: '#8A2BE2'
+  Color.create name: 'sienna', hex_value: '#A0522D'
 end
 
 if Parrot.table_exists? && Parrot.count == 0
-  Parrot.create! name: 'Billy', sex: 'male',   age: 1,  color: Color.find_by_name('red')
-  Parrot.create! name: 'Willy', sex: 'male',   age: 15, color: Color.find_by_name('green')
-  Parrot.create! name: 'Kelly', sex: 'female', age: 19, color: Color.find_by_name('green')
+  Parrot.create! name: 'Billy',  sex: 'male',   age: 6,  color: Color.find_by_name('red')
+  Parrot.create! name: 'Willy',  sex: 'male',   age: 3,  color: Color.find_by_name('violet'), tribal: true
+  Parrot.create! name: 'Kelly',  sex: 'female', age: 16, color: Color.find_by_name('green'), tribal: true
+  Parrot.create! name: 'Trelly', sex: 'male',   age: 28, color: Color.find_by_name('green'), tribal: true
+  Parrot.create! name: 'Fred',   sex: 'male',   age: 14, color: Color.find_by_name('gold')
+
+  mark  = Parrot.create! name: 'Mark',  sex: 'male',   age: 93, color: Color.find_by_name('orange'), tribal: true
+  wenny = Parrot.create! name: 'Wenny', sex: 'female', age: 48, color: Color.find_by_name('orange'), tribal: true
+  marry = Parrot.create! name: 'Marry', sex: 'female', age: 36, color: Color.find_by_name('sienna'), mother_id: wenny.id, father_id: mark.id, tribal: true
+
+  john = Parrot.create! name: 'John',  sex: 'male',   age: 85, color: Color.find_by_name('blue'), tribal: true
+  katy = Parrot.create! name: 'Katy',  sex: 'female', age: 56, color: Color.find_by_name('blue'), tribal: true
```

Comment:
Line is too long. [110/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.